Source/FluidOscServer.cpp

```diff
@@ -291,8 +291,14 @@ void FluidOscServer::clearMidiClip(const juce::OSCMessage& message) {
 }
 
 void FluidOscServer::insertMidiNote(const juce::OSCMessage& message) {
-    if (!selectedMidiClip) return;
-    if (message.size() < 3) return;
+    if(!selectedAudioTrack){
+        std::cout << "Cannot load Audio Track: Must select Audio Track before inserting" << std::endl;
```

Comment:
Should probably be "Cannot insert midi note: no audio track selected" or similar